tests/test_lane.py

```diff
@@ -1577,6 +1633,151 @@ def for_audiences(*audiences):
                  non_gutenberg_children.id]),
             set(for_audiences()))
 
+    def test_customlist_filter_clauses(self):
```

Comment:
This test was moved from TestLane to TestWorkList and I did have to change it a little bit to create WorkLists instead of Lanes.